common/rc

```diff
@@ -140,3 +143,101 @@ _get_pci_dev_from_blkdev() {
 	readlink -f "$TEST_DEV_SYSFS/device" | \
 		grep -Eo '[0-9a-f]{4}:[0-9a-f]{2}:[0-9a-f]{2}\.[0-9a-f]'
 }
+
+_notrun()
+{
+	echo "$*" > $seqres.notrun
+	echo "$seq not run: $*"
+	rm -f ${RESULT_DIR}/require_test*
+	rm -f ${RESULT_DIR}/require_scratch*
+
+	status=0
+	exit
+}
+
+_fail()
```

Comment:
This also won't work in blktests. If helpers are calling it, they should just print an error and return 1 instead, and tests usually shouldn't be exiting.